Makefile

```diff
@@ -15,8 +12,11 @@ build: clean_pyc
 lint-diff:
 	git diff upstream/master --name-only -- "*.py" | xargs flake8
 
+black:
+	black . --exclude '(asv_bench/env|\.egg|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist)'
```

Comment:
Is this to speed up (to prevent black from traversing those dirs)? as some of those dirs should in principle never contain .py files (eg build, .git or .mypy_cache)

Reply:
no these are black defaults for exclude except for the asv env one
i am unsure whether the defaults are there if u override

Reply:
Ah, OK, I see. Yeah, I think it is not additive, so we need to do this .. (for this it would be nice to be able to put this in a pyproject.toml, maybe we should look at that again)